docs/pages/machine-workload-identity/machine-id/deployment/kubernetes-oidc.mdx

```diff
@@ -64,8 +64,8 @@ possible.
 
 ## Prerequisites
 
-(!docs/pages/includes/edition-prereqs-tabs.mdx!)
-
+- A running Teleport cluster, version 18.1.5 or above.
```

Comment:
I'm planning to tweak the specific numbers in the v17 backport to reflect the actual version the backport was released in.

Reply:
It's possible to use the partial with a version restriction, e.g., https://github.com/gravitational/teleport/blame/426f9f4198a3934a59de0866ff6bd941d51827a7/docs/pages/enroll-resources/agents/oracle.mdx#L24